### ISSUE TITLE
Gewichtung der Antworten

### ISSUE THREAD

**Issue by binfalse** [wahlbilanz/DeinWal.de#13]
Hab jetzt schon oefter gelesen, dass Leute sich eine Gewichtung wuenschen -- "wie beim Wahl-O-Mat".. (eg https://www.reddit.com/r/de/comments/6ug9ui/deinwalde_wahlomat_mit_bisherigen_abstimmungen/dlsuwfm/)

Hatten wir ja auch lange nachgedacht.. Vielleicht implementiert das noch jemand? Hat jemand ein proposal fuer ein gewichtetes Rechenmodel?

**Comment by tomthe** [comment]
3 Sachen: UI, Rechenmodell, Implementierung

UI: eine neue, schmale Spalte "Gewichtung". Standardmäßig ist alles normal, also 1-fach gewichtet. Dargestellt durch... nichts? Ein Klick: Sternchen = 2-fache gewichtung. Noch ein Klick: Mülltonne/Kreuz/?? 0-Fache Gewichtung

Rechenmodell: wie bisher, bloß das bei der Mittelwertberechnung 1-fach oder 2-fach oder 0-fach gewichtet wird.

Implementierung: tja, mit Testen dauert das wieder ein paar Stunden... Freiwillige?

**Comment by binfalse** [comment]
Ich glaub das UI in der Auswertung geht so nicht so einfach. Auf mobile ist da echt wenig platz, eg:

<img src="https://scratch.binfalse.de/deinwal-auswertung-mobile-alles-ja.jpg" alt="auswertung mit alles ja angeklickt" width="200px">

Das ist jetzt schon so voll, und nervt, dass die carets manchmal in die naechste Zeile rutschen. Wo soll da noch Sternchen/Muelltonne hin? Und eine ganze Spalte braucht echt viel Platz..
Vielleicht ist das dann sowieso komisch, weil man ja dann auch schon sieht wer wie du gestimmt hat -- und sich das ergebnis "hinbastelt"..

Vielleicht eher ein sternchen waehrend des Quiz'? Also irgendwo direkt bei der Frage? Aber hab auch keine gute Idee wie/wo genau, ohne das ganze noch mehr noisy zu machen...

**Comment by tomthe** [comment]
Ja, du hast in beiden punkten Recht. Zu wenig Platz in der MobilAnsicht könnte man umgehen, indem man es erst nach dem Aufklappen anzeigt. Dann bleibt das Problem mit dem Ergebnis-zurechtbasteln. 

Noch eine Lösung wäre, wie beim Wahl-O-Mat eine extra Seite nur für die Gewichtung vor der Ergebnis Seite zu erstellen. Das würde aber den “Flow“ stören, insbesondere für Leute, die nicht gewichten wollen. Dagegen könnte man aber oben einen Button hinsetzen: “ohne Gewichtung direkt zu den Ergebnissen“

Einfacher und unobstriver ist das platzieren direkt bei jeder Frage. Wo dort? In der Nähe von oder neben ja/ Nein/ enthalten? Rechts neben “mehr Informationen“?

**Comment by jzaefferer** [comment]
Eine Null-Gewichtung verstehe ich nicht - reicht es da nicht, die Frage unbeantwortet zu lassen?

Doppelte Gewichtung könnte beim Quiz durch ein Sternchen o.ä. dargestellt werden. Bei der Auswertung durch Formatierung (fett oder kursiv?), wenn kein Platz mehr ist.

**Comment by jzaefferer** [comment]
Es könnte auch spannend zu sein, kontroverse Fragen hervor zu heben. Hängt ein wenig mit #31 zusammen.

**Comment by binfalse** [comment]
Stimmt, Null-Gewichtung macht wenig Sinn... :)

Sternchen (oder Ausrufezeichen?) bei der Frage ist also wahrscheinlich die beste Loesung. Irgendwelche Meinungen wo das noch hin soll?^^

also vor/hinter die Frage? Oder mit zu den ja/nein/enthalten buttons? -> aber die brechen dann auf mobil bestimmt wieder so bloed um...
